dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowProcessor.java

```diff
@@ -87,19 +87,21 @@ public WorkflowProcessor(Contentlet contentlet, User firingUser) {
 
 		try {
 			this.user = firingUser;
-			scheme = getWorkflowAPI().findSchemeForStruct(contentlet.getStructure());
+			WorkflowStep step = null;
+			List<WorkflowStep> steps = getWorkflowAPI().findStepsByContentlet(contentlet);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "steps" which hides the field declared at line 20. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)